CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## v0.129.1, 21 December 2020
+
+- Bump phpstan/phpstan from 0.12.62 to 0.12.63 in /composer/helpers/v2
+- Bump friendsofphp/php-cs-fixer in /composer/helpers/v2
+- Composer: Verify if composer name is compatible with v2
+- Python: Upgrade Python version to 3.9.1 (@ulgens)
+- Bump npm from 6.14.9 to 6.14.10 in /npm_and_yarn/helpers
+- Bump eslint from 7.15.0 to 7.16.0 in /npm_and_yarn/helpers
+- Bump friendsofphp/php-cs-fixer in /composer/helpers/v1
+- Sanitize creds from DependabotError
+- Bump composer/composer from 1.10.16 to 1.10.19 in /composer/helpers/v1
+- Bump phpstan/phpstan from 0.12.62 to 0.12.63 in /composer/helpers/v1
```

Comment:
Question: should we track `composer/v1` and `composer/v2` separately? I'm not clear on the value (other than that's what Dependabot forces due to lack of grouping).

Reply:
There's a small difference in some of the dependencies between v1 and v2. I don't think we need to list them separately in the changelog, happy to simplify here if that's what you meant?